modules/sdk/gradle-plugins-app-javadoc-builder/README.markdown

```diff
@@ -0,0 +1,59 @@
+# App Javadoc Builder Gradle Plugin
+
+The App Javadoc Builder Gradle plugin allows you to generate a combined HTML API
+documentation for an application that spans different subprojects, each one
```

Comment:
I mean a single Javadoc "package" for a series of modules, with a single `index.html`.